application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java

```diff
@@ -64,7 +64,6 @@ public class BeanConfigurator
     private final AtomicReference<Security.CqlSecurity> cqlSecurity = new AtomicReference<>();
     private final AtomicReference<Security.JmxSecurity> jmxSecurity = new AtomicReference<>();
     private final ConfigRefresher configRefresher;
-    private final String ecChronosID;
```

Comment:
Idk about removing it, because we need to generate this ans pass for EccNodeMetadata if the instance is new, I think here we need to have some validation using the EccNodeMetadata, generate the unique id, and them passing for EccNodesSync